src/index.js

```diff
@@ -243,8 +243,8 @@ function bindRouteSet (routes = [], controller, server, options = {}) {
  *
  * @param {object} cache - a koop database adapter
  */
-Koop.prototype._registerCache = function (Cache) {
-  this.cache = new Cache()
+Koop.prototype._registerCache = function (Cache, options) {
+  this.cache = (typeof options !== 'undefined') ? new Cache(options) : new Cache()
```

Comment:
I think you can just call new Cache(options) always

Reply:
excellent.

Reply:
I think he means to just do: `this.cache = new Cache(options)`.  You don't need the check.